rtf_vis_tool/package-lock.json

```diff
@@ -0,0 +1,17982 @@
+{
+  "name": "rtf-intro",
+  "version": "0.1.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
```

Comment:
there are a lot of dependencies :-) 

Do we need every one of them? What are they fore?

Reply:
to be honest, a lot of the stuff in package-lock.json comes auto generated when I initialize my react application. I'll look into if and how I can safely clean this file up. Because you're right, it's extremely long...

Reply:
Was able to clean some of it up, but this package-lock.json file will have to be checked into Github to ensure all users are using the same package versions.